src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int randomColor = random.nextInt(Colors.values().length);
+        return Colors.values()[randomColor].name();
```

Comment:
Instead of calling `Colors.values()` twice which creates a new array each time, you should store the result into a variable and reuse it.